app/views/pages/amends/outcomeAmended.scala.html

```diff
@@ -0,0 +1,101 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.AmendResultDisplayModel
+@import models.amendModels.AmendsGAModel
+@import play.api.i18n._
+@import enums.ApplicationType
+@import utils.Constants
+
+@this(
+layout: Layout,
+resultAmendDetails: playHelpers.templates.resultAmendDetails
+)
+
+@(model: AmendResultDisplayModel)(implicit request: Request[_], messages: Messages)
+
+@layout(
+pageTitle = messages("pla.activeAmend.title")
```

Comment:
I don't think it is a good idea to re-use messages from other pages. Unintentionally, they may be removed or changed and introduce wrong content for the other page. Besides, It is easier to create new ones now, rather than when a ticket to remove/change them appears.